srcpkgs/retro-gtk/template

```diff
@@ -0,0 +1,37 @@
+# Template file for 'retro-gtk'
+pkgname=retro-gtk
+version=0.16.0
+revision=1
+build_style=meson
+hostmakedepends="glib-devel pkg-config vala"
+makedepends="gtk+3-devel libepoxy-devel libglib-devel pulseaudio-devel"
+short_desc="The GTK+ Libretro frontend framework"
+maintainer="Rasmus Thomsen <rasmus.thomsen@protonmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://gitlab.gnome.org/GNOME/retro-gtk"
+distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
+checksum=8c9b4ec9d6aa6b9db088a09706c0dd3de41437ff3a46f46566a47911efa7a627
+
+if [ "$CROSS_BUILD" ]; then
```

Comment:
maybe have it as a build_option, one day we might fix gobject-introspection and just flipping a build_option switch will be nice